lib/manager/homebrew/extract.spec.ts

```diff
@@ -88,32 +88,6 @@ describe('manager/homebrew/extract', () => {
       expect(res.deps[0].skipReason).toBe('unsupported-url');
       expect(res).toMatchSnapshot();
     });
-    it('skips if invalid url extension', () => {
-      const content = `
-          class Ibazel < Formula
-          desc 'IBazel is a tool for building Bazel targets when source files change.'
-          homepage 'https://github.com/bazelbuild/bazel-watcher'
-          url "https://github.com/bazelbuild/bazel-watcher/archive/v0.8.2.not_tar.not_gz"
-          sha256 '26f5125218fad2741d3caf937b02296d803900e5f153f5b1f733f15391b9f9b4'
-          end
-      `;
-      const res = extractPackageFile(content);
-      // FIXME: explicit assert condition
-      expect(res).toMatchSnapshot();
-    });
-    it('skips if invalid url version', () => {
-      const content = `
-          class Ibazel < Formula
-          desc 'IBazel is a tool for building Bazel targets when source files change.'
-          homepage 'https://github.com/bazelbuild/bazel-watcher'
-          url "https://github.com/bazelbuild/bazel-watcher/archive/vInvalid.version.2.tar.gz"
-          sha256 '26f5125218fad2741d3caf937b02296d803900e5f153f5b1f733f15391b9f9b4'
-          end
-      `;
-      const res = extractPackageFile(content);
-      // FIXME: explicit assert condition
-      expect(res).toMatchSnapshot();
-    });
```

Comment:
These tests were relying on internal version validation, but now we're using centralized post-extract validation. Currently, there is no any `skip-reason` in snapshots.